method/link.js

```diff
@@ -84,6 +84,50 @@ module.exports = function Link (ssb) {
         }),
         pull.collect(cb)
       )
+    },
+    findSubgroupByGroupId (groupId, cb) {
+      if (!isGroup(groupId)) return cb(new Error(`findSubgroupByGroupId expected a groupId, got ${groupId} instead.`))
+
+      const query = [{
+        $filter: {
+          value: {
+            content: {
+              type: 'link/group-subgroup',
+              parent: groupId,
+              tangles: {
+                link: { root: null, previous: null }
+              }
+            }
+          }
+        }
+      }]
+
+      pull(
+        ssb.query.read({ query }),
```

Comment:
this should be using backlinks, not query - backlinks is optimised for finding the small number of things which directly reference a particular cypherlink. 
Query is similar, except you add `dest: groupId` at the same level as `value` I think

:fire: (must change)

Reply:
Should i fix this for the function above `findGroupByFeedId` as thats where i copied this from?

Reply:
Fixed both